lib/index.test.js

```diff
@@ -358,36 +419,37 @@ describe('lib/index', () => {
     expect(logger.info).toHaveBeenNthCalledWith(4, `npm i foo1@1.0.0-alpha.2 `);
   });
 
-  it('should not upgrade when no version available for major version of pre-release', async () => {
+  it.skip('should not upgrade when no version available for major version of pre-release', async () => {
```

Comment:
Is this the test that you could not get working? (I think that was mentioned), just making sure the skip is meant to be there! 👍

Reply:
this is intentional, this tests need to be reviewed

Reply:
Yep, just making sure!